contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -37,11 +23,15 @@ RUN \
 # Build dependencies
 #
 apt-get update && \
-apt-get install -y --no-install-recommends make git curl ca-certificates xz-utils \
+apt-get install -y --no-install-recommends \
+# Base runtime-build dependencies
+  make curl ca-certificates xz-utils \
 # Additional iproute2 build dependencies
   gcc git pkg-config bison flex build-essential \
 # Additional bpftool dependencies
-  python3 && \
+  python3 gcc-multilib \
```

Comment:
Do we still need `gcc-multilib` with the recent changes in the `bpftool` build?

Reply:
Good point, nope. I'm also thinking to shrink down more from the runtime dependencies we install there.